tests/publiccloud/create_aistack_env.pm

```diff
@@ -1,4 +1,4 @@
-# SUSE's openQA tests
+)# SUSE's openQA tests
```

Comment:
A tidy before push always useful: remove this leftover before next commit and VR.